### DISCUSSION TITLE
检查文件校验和，文件校验结果不正确，

### DISCUSSION THREAD

**Discussion by zhouHnsq** [he3-app/he3-feedback#189] [Bug Report]
只勾选sha256没问题，但是多选一个校验，就会造成sha256计算结果不同。比如：只勾选sha256，sha256计算结果正确。勾选md5和sha256，重新上传同一个文件，此时sha256输出的结果是不对的

**Comment by lyzhang1999** [comment]
感谢反馈，我已经创建了 issues：https://github.com/he3-app/he3-feedback/issues/190

你可以关注该 issues 获取最新状态。
